packages/contracts-bedrock/package.json

```diff
@@ -25,7 +25,7 @@
     "test": "yarn build:differential && forge test",
     "coverage": "yarn build:differential && forge coverage",
     "coverage:lcov": "yarn build:differential && forge coverage --report lcov",
-    "gas-snapshot": "forge snapshot --no-match-test 'differential|fuzz'",
+    "gas-snapshot": "yarn build:differential && forge snapshot --no-match-test 'differential|fuzz'",
```

Comment:
why do we need this back in?

Reply:
The finalizeWithdrawal tests rely on those utils existing, even for the non-differential tests. It's a pretty quick part of the task so not a big deal.